packages/central-server/package.json

```diff
@@ -13,14 +13,14 @@
   "types": "dist/index.d.ts",
   "scripts": {
     "build": "rm -rf dist && run-p -c \"build:* {@}\" --",
-    "build:src": "babel src -s --out-dir dist --source-maps --ignore \"**src/database/migrations\",\"src/tests/**\" --copy-files --no-copy-ignored --config-file \"../../babel.config.json\"",
-    "build:types": "tsc",
-    "lint": "eslint --ignore-path ../../.gitignore .",
+    "build:src": "yarn package:build:js --ignore \"**src/database/migrations\",\"src/tests/**\" --copy-files --no-copy-ignored",
+    "build:types": "yarn package:build:types",
+    "lint": "yarn package:lint:js",
     "lint:fix": "yarn lint --fix",
-    "prestart": "npm run -s build",
+    "prestart": "yarn build",
     "start": "node dist",
-    "start-dev": "../../scripts/bash/backendStartDev.sh 9999",
-    "start-verbose": "LOG_LEVEL=debug npm run start-dev",
+    "start-dev": "yarn package:start:backend-start-dev 9999",
+    "start-verbose": "LOG_LEVEL=debug yarn start-dev",
     "test": "yarn workspace @tupaia/database check-test-database-exists && DB_NAME=tupaia_test mocha",
```

Comment:
Packages which use non-standard scripts e.g. mocha declare them manually and have those dependencies explicitly specified.